swagger.go

```diff
@@ -11,21 +11,28 @@ import (
 	"github.com/swaggo/swag"
 )
 
+// Config stores ginSwagger configuration variables.
+type Config struct {
+	//The url pointing to API definition (normally swagger.json or swagger.yaml).
+	URL string
+}
+
 // WrapHandler wraps `http.Handler` into `gin.HandlerFunc`.
-func WrapHandler(h *webdav.Handler) gin.HandlerFunc {
+func WrapHandler(config *Config, h *webdav.Handler) gin.HandlerFunc {
```

Comment:
It seems safer to prepare another function.

Reply:
I think feel good is revert and create another function.

Reply:
I am syncing  up with swag code. sorry about bother.

Reply:
I see.
I was concerned about was the impact on users already using `WrapHander` .
So I think create another interface. eg: config setable and default config (`WrapHander`) .

Reply:
I agree. could you help add this feat?

Reply:
@pei0804

Reply:
OK I do it.
Before that, I would like to revert this PR once to minimize the impact.

Reply:
ok